source/releasenotes/2024-04-03-new-relic-agent.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Upcoming New Relic PHP Agent update changes Drupal hooks default behavior [Action Required]
+published_date: "2024-04-03"
+categories: [action-required, tools-apis]
+---
+
+We are thrilled to announce that on April 23, 2024, we will upgrade the New Relic PHP agent to the latest version (v10.19.0.9). This update is part of our ongoing commitment to enhancing your experience with Pantheon and ensuring that our offerings meet your evolving needs.
+
+By staying up-to-date with the latest agent, you will benefit from improved performance and enhanced troubleshooting capabilities for your WordPress and Drupal instances. This means smoother experiences for your site visitors and more efficient workflows for you. This new version also fully supports PHP 8.3, providing you with the latest features and compatibility.
+
+[Learn More about New Relic®](https://docs.pantheon.io/guides/new-relic), real-time performance monitoring for your Pantheon web applications. 
+
+## Action Required: Default behavior of New Relic Drupal Hooks is changing
+
+After the release of this agent update, Pantheon will no longer support Drupal Hooks reporting by default. However, the functionality is still available. If this is crucial for troubleshooting your sites, please add these lines to your site’s pantheon.yml file to enable this reporting:
+
+```
+new_relic:
```

Comment:
@scottbuscemi can you add an explanation for why it's not on by default?

Reply:
I believe cost is the reason it's not on by default.